composer.json

```diff
@@ -46,10 +46,11 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require-dev": {
-        "behat/behat": "^3.6",
+        "php": "^8.0",
```

Comment:
Do we require PHP 8 here? How does that play with the DDEV PHP 7.4 sandbox?

Reply:
We need it for require-dev. It doesn't affect the sandbox